src/applications/ezr/utils/helpers/form-config.js

```diff
@@ -47,6 +48,41 @@ export function includeTeraInformation(formData) {
   return formData.hasTeraResponse;
 }
 
+export function veteranBornBetween(formData, date1, date2) {
+  return isWithinInterval(new Date(formData?.veteranDateOfBirth), {
+    start: new Date(date1),
+    end: new Date(date2),
+  });
+}
+
+export function canVeteranProvideAgentOrangeResponse(formData) {
+  return (
+    includeTeraInformation(formData) &&
+    veteranBornBetween(formData, '1900-01-01', '1965-12-31')
```

Comment:
These kinds of things are where Typescript is really helpful, since we could type these params to be a string vs date and catch these really fast. Nothing to change here, just wanted to mention it 🙂

Reply:
Yes, totally agree! Thank you for taking another look. I really appreciate it! 😄